katas/juan-molina/Katas/Week-01/FibonacciOcurrences/FibonacciOcurrences.md

```diff
@@ -0,0 +1,27 @@
+Fibonacci numbers are generated by setting F0 = 0, F1 = 1, and then using the formula:
```

Comment:
Please change all markdown file names to README.md so that Github recognizes and displays them on each kata folder view

Reply:
Also, you are missing the links to katas on each README.md file